### PR TITLE
Update page title for better search results

### PR DESCRIPTION
**Pull request by dobooth**
### Description

Magento search results list page titles.
There is another topic called "Versioning" so I updated to the title to differentiate.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: docs/versioning.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ This table lists the version of the MFTF that was released with a particular ver
 
 |Magento version| MFTF  version|
 |---|---|
+| 2.3.2 | 2.4.2 |
```
Comment (KevinBKozan) [inline docs/versioning.md]: From what I can tell, Magento `2.3.2` actually has `2.3.14`. Am I misunderstanding something?

https://github.com/magento/magento2ce/blob/2.3.2/composer.json#L89

Comment (dobooth) [reply docs/versioning.md]: Good!

File: docs/versioning.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Versioning
+# MFTF versioning
```
Comment (dshevtsov) [inline docs/versioning.md]: Change to `MFTF versioning schema` to be consistent with Magento documentation in terminology.

Comment (dobooth) [reply docs/versioning.md]: Fixed.

**Comment by dobooth** [comment]
Moved to other PR.

**Comment by dshevtsov** [comment]
@dobooth please link the other PR here.

**Pull request closed without merging**